README.md

```diff
@@ -1,14 +1,58 @@
-# Prometheus Integration Pack
+# prometheus integration pack v0.2.2
+
+> Interact with Prometheus HTTP API
+Daniel Chamot <daniel@nullkarma.com>
+
+### Contributors
+- Carlos <nzlosh@yahoo.com>
```

Comment:
It looks a bit weird that `Contributors` section comes first in the README. What's the reason behind this?

Do we want to update the README every time for anyone who're making a PR? Do we also want to include old contributors who made PRs in the past against this repo?  How about the original pack author defined in the `pack.yml`?
I found this practice similar to outdated one in the open-source world when people added their credits & copyrights in every file they edited.

I understand this is generated from the `pack.yml`. What matters more to me who's maintaining the pack on a regular basis. @nzlosh Do you want to be a maintainer for this pack? If so, let us know and we'll add you permissions. Also adding Codeowners would be a good fit for this: https://github.com/StackStorm-Exchange/exchange-incubator/issues/152

Reply:
I saw in another pack (I can't recall which one now) where contributors were acknowledge for their PRs in the readme.  I personally find it a nice sign of respect and appreciation of the effort others have made to give time and effort to the pack to acknowledge that fact in the readme.

People who commit can update the pack.yaml themselves to choose to be included in the contributors section.  If it's an outdated practice then the template can be updated to not include it or move it the bottom of the readme.

The pack2md project could be updated to process the Codeowners file and include it in the README.md as well.

I currently actively use the prometheus pack and would be fine with becoming a maintainer.

Reply:
At minimum, having this block as a first README section feels weird and never seen before.
Instead of providing the most useful information for the potential pack consumer it tries to promote the people names which I believe is not the main goal of this repository.

Having the list of Github users in the Readme who contributed to the pack at least once would be out of place. Having the list of Maintainers (with contacts) who are supporting the pack as on-going basis and keeping that list up to date by adding/removing active/inactive maintainers with their github and email contacts sounds great.